### ISSUE TITLE
Add English Multilingual Librispeech

### ISSUE THREAD

**Issue by yaya-sy** [yaya-sy/SpeechAya#7]
Extract the English subset in the Multilingual Librispeech dataset (https://huggingface.co/datasets/facebook/multilingual_librispeech). The resulting data must look like this:
```
{
   'audio': the audio file,
   'text": the transcription of the audio
}
```
Then, you can push the dataset to HuggingFace.

**Comment by mariadhakal** [comment]
@yaya-sy It seems like this dataset doesn't have English data.

**Comment by yaya-sy** [comment]
@mariadhakal, you're right! For English, we have a separate dataset for Librispeech: [openslr/librispeech_asr](https://huggingface.co/datasets/openslr/librispeech_asr). We need the 'clean' subset and the 'train-360' split.

<img width="1409" alt="Capture d’écran 2024-08-03 à 23 34 26" src="https://github.com/user-attachments/assets/4dcd5bdd-861a-4161-a938-4740cbd847ab">

**Comment by mariadhakal** [comment]
Thanks!

**Comment by mariadhakal** [comment]
I have uploaded the dataset. It was my first time doing such a task. Can you please check and let me know if everything is okay? https://huggingface.co/datasets/mariadhakal/dataset_librispeech_english_clean_train360

**Comment by yaya-sy** [comment]
Thank you! I see a list of floats in the 'audio' column in the pushed data. Actually, we need an 'audio dataset' on Hugging Face. See here: https://huggingface.co/docs/datasets/audio_dataset. For the LibriSpeech dataset you're working on, I think we can achieve our goal by removing all columns except for 'audio' and 'text.'

And since we don't have as much data for English, in addition to `train.clean.360`, I think we will also need to add `train.clean.100` and `train.other.500`. Maybe @abheesht17 can look into that.
